### PR TITLE
Padding bytes should be ignored when parsing RTPPacket

### PR DESCRIPTION
**Pull request by pedoc**
Padding bytes are redundant and will cause the video decoder to fail to decode.

### CODE REVIEW

File: src/net/RTP/RTPHeader.cs [hunk 1]
```diff
@@ -78,26 +81,34 @@ public RTPHeader(byte[] packet)
...
             PaddingFlag = (firstWord >> 13) & 0x1;
             HeaderExtensionFlag = (firstWord >> 12) & 0x1;
             CSRCCount = (firstWord >> 8) & 0xf;
+
             MarkerBit = (firstWord >> 7) & 0x1;
             PayloadType = firstWord & 0x7f;
 
+            int headerExtensionLength = 0;
+            // ReSharper disable once InconsistentNaming
             int headerAndCSRCLength = 12 + 4 * CSRCCount;
```
Comment (sipsorcery) [inline src/net/RTP/RTPHeader.cs:95]: Can these resharper warnings be done in a config file somewhere? It doesn't suit all of us to have these comments scattered throughout the code.

**Review by sipsorcery — CHANGES_REQUESTED** [review]

**Comment by sipsorcery** [comment]
Thanks for the PR. Can't say I've come across RTP padding but you're right if it's in the wild it could easily cause probs for the video and audio decoders

File: src/net/RTP/RTPHeader.cs (hunk 1 above)
Comment (pedoc) [reply src/net/RTP/RTPHeader.cs:94]: sure,solved

**Comment by sipsorcery** [comment]
I reverted the PR due to the unit test failure below:

`[xUnit.net 00:00:00.74]     SIPSorcery.Net.UnitTests.RTPHeaderUnitTest.ParseRawRtpWithExtensionTest [FAIL]
  Failed SIPSorcery.Net.UnitTests.RTPHeaderUnitTest.ParseRawRtpWithExtensionTest [< 1 ms]
  Error Message:
   System.ArgumentException : Source array was not long enough. Check the source index, length, and the array's lower bounds. (Parameter 'sourceArray')
  Stack Trace:
     at System.Array.Copy(Array sourceArray, Int32 sourceIndex, Array destinationArray, Int32 destinationIndex, Int32 length, Boolean reliable)
   at System.Array.Copy(Array sourceArray, Int32 sourceIndex, Array destinationArray, Int32 destinationIndex, Int32 length)
   at SIPSorcery.Net.RTPPacket..ctor(Byte[] packet) in /home/runner/work/sipsorcery/sipsorcery/src/net/RTP/RTPPacket.cs:line 41
   at SIPSorcery.Net.UnitTests.RTPHeaderUnitTest.ParseRawRtpWithExtensionTest() in /home/runner/work/sipsorcery/sipsorcery/test/unit/net/RTP/RTPHeaderUnitTest.cs:line 165`

**Comment by pedoc** [comment]
> I reverted the PR due to the unit test failure below:
> 
> `[xUnit.net 00:00:00.74] SIPSorcery.Net.UnitTests.RTPHeaderUnitTest.ParseRawRtpWithExtensionTest [FAIL] Failed SIPSorcery.Net.UnitTests.RTPHeaderUnitTest.ParseRawRtpWithExtensionTest [< 1 ms] Error Message: System.ArgumentException : Source array was not long enough. Check the source index, length, and the array's lower bounds. (Parameter 'sourceArray') Stack Trace: at System.Array.Copy(Array sourceArray, Int32 sourceIndex, Array destinationArray, Int32 destinationIndex, Int32 length, Boolean reliable) at System.Array.Copy(Array sourceArray, Int32 sourceIndex, Array destinationArray, Int32 destinationIndex, Int32 length) at SIPSorcery.Net.RTPPacket..ctor(Byte[] packet) in /home/runner/work/sipsorcery/sipsorcery/src/net/RTP/RTPPacket.cs:line 41 at SIPSorcery.Net.UnitTests.RTPHeaderUnitTest.ParseRawRtpWithExtensionTest() in /home/runner/work/sipsorcery/sipsorcery/test/unit/net/RTP/RTPHeaderUnitTest.cs:line 165`

#614 Solved this problem

**Pull request merged**